addons/core/translations/resources/en-us.yaml

```diff
@@ -560,6 +560,22 @@ role:
     description: Grants are permissions which allow roles to take actions and access resources.
     actions:
       create: New Grant
+  scope:
+    title: Scope
+    title_plural: Scopes
+    description: Role grant sets can be applied to multiple descendants underneath this scope.
```

Comment:
It looks like we're using this description for any role page, but this isn't applicable if it's org (children only) or project (no descendants or children) scopes right?

Reply:
Oh that is a good point. I can ask if we should use more generic language here or we could change the language depending on the scope. Good catch.

Reply:
This has been updated.